README.md

```diff
@@ -34,54 +34,72 @@ A grouping element to hold other elements.
 #### Properties
 
 ##### category.copy
-Get an array element of all child elements with the element name `copy`. This property is **ready-only**.
+Get an array element of all child elements with the element name `copy`. This property is **read-only**.
 
 ```js
 let copy = category.copy;
 ```
 
 ##### category.dataStructures
-Get an array element of all child elements with the element name `category` and a class name `dataStructures`. This property is **ready-only**.
+Get an array element of all child elements with the element name `category` and a class name `dataStructures`. This property is **read-only**.
 
 ```js
 let dataStructures = category.dataStructures;
 ```
 
 ##### category.resources
-Get an array element of all child elements with the element name `resource`. This property is **ready-only**.
+Get an array element of all child elements with the element name `resource`. This property is **read-only**.
 
 ```js
 let resources = category.resources;
 ```
 
 ##### category.resourceGroups
-Get an array element of all child elements with the element name `category` and a class name `resourceGroup`. This property is **ready-only**.
+Get an array element of all child elements with the element name `category` and a class name `resourceGroup`. This property is **read-only**.
 
 ```js
 let groups = category.resourceGroups;
 ```
 
 ##### category.scenarios
-Get an array element of all child elements with a class name `scenario`. This property is **ready-only**.
+Get an array element of all child elements with a class name `scenario`. This property is **read-only**.
 
 ```js
 let scenarios = category.scenarios;
 ```
 
 ##### category.transitions
-Get an array element of all child elements with the element name `transition`. This property is **ready-only**.
+Get an array element of all child elements with the element name `transition`. This property is **read-only**.
 
 ```js
 let transitions = category.transitions;
 ```
 
 ##### category.transitionGroups
-Get an array element of all child elements with the element name `category` and a class name `transitions`. This property is **ready-only**.
+Get an array element of all child elements with the element name `category` and a class name `transitions`. This property is **read-only**.
 
 ```js
 let groups = category.transitionGroups;
 ```
 
+##### category.authSchemes
+Get an array element of all child element with element name equal to one of the names given below. This property is **read-only**.
+
+* Basic Authentication Scheme
+* Token Authentication Scheme
+* Oauth2 Scheme
```

Comment:
I know this is how it is in the Refract spec, but I think we should start moving to "OAuth2" spelling as per the OAuth spec before we end up having it "Oauth2" everywhere and can't change.